### ISSUE TITLE
Require toggle/password for private entries

### ISSUE THREAD

**Issue by vvirtues** [MALSync/MALSync#2452]
### Suggestion

Say I have something set as private and I don't want the extension to openly show it.  You can implement a password/PIN before seeing the content (or requiring direct search) or have a hidden toggle somewhere. Would be nice to have

**Comment by vvirtues** [comment]
https://github.com/MALSync/MALSync/issues/414#issuecomment-2155448338

**Comment by vvirtues** [comment]
still waiting for this

**Comment by vvirtues** [comment]
Still waiting on this
